lua/harpoon/buffer.lua

```diff
@@ -73,6 +73,20 @@ function M.setup_autocmds_and_keymaps(bufnr)
         "<Cmd>lua require('harpoon.buffer').run_select_command()<CR>",
         {}
     )
+    vim.api.nvim_buf_set_keymap(
+        bufnr,
+        "n",
+        "<C-x>",
+        "<Cmd>lua require('harpoon.buffer').run_select_command({ split = true })<CR>",
+        {}
+    )
```

Comment:
Same as before (github doesn't let you comment on lines that have not changed)
```lua
    vim.keymap.set(
        "n",
        "q",
        function() require("harpoon.buffer").run_toggle_command("q") end,
        { silent = true, buffer = bufnr }
    )
    vim.keymap.set(
        "n",
        "<ESC>",
        function() require("harpoon.buffer").run_toggle_command("Esc") end,
        { silent = true, buffer = bufnr }
    )
    vim.keymap.set(
        "n",
        "<CR>",
        function() require("harpoon.buffer").run_select_command() end,
        { buffer = bufnr }
    )
    vim.keymap.set(
        "n",
        "<C-x>",
        function() require("harpoon.buffer").run_select_command({ split = true }) end,
        { buffer = bufnr }
    )
```

Reply:
We could also just use callback option in `vim.api.nvim_buf_set_keymap`, but oke